### DISCUSSION TITLE
Tool Selection

### DISCUSSION THREAD

**Discussion by mtewatia2020** [pyqtgraph/pyqtgraph#2262] [General]
Scenario- A medium size business reaches out to you to assist them in Data management and create a solid Analytical platform for their multiple teams. What would be the framework to 1. Select tools for database, analytics and so on 2. What skills will be required to deliver on this task

**Comment by j9ac9k** [comment]
Hi @mtewatia2020 I think you may have posted this in the wrong place, as what you are discussing has little relevance/overlap with pyqtgraph.  This is not the correct place for this type of inquery, as it has nothing to do with pyqtgraph.
